action.py

```diff
@@ -244,7 +248,7 @@ def check_commit(urepo, ubranch, target, sha, merge):
 
         match = re.search(regex, body, re.MULTILINE)
         if not match:
-            die(f'{sha}: fromlist commit missing an Upstream PR reference')
+            die(f'{sha}: fromlist commit missing an "Upstream PR #:" reference')
```

Comment:
```suggestion
            die(f'{sha}: fromlist commit message missing an "Upstream PR #:" reference'
                  f'See [doc](https://nordicsemi.atlassian.net/wiki/spaces/NCS/pages/108201225/Pull+Requests#fromlist-ref) for explanation.')
           
            
```

Reply:
The reference to the Development Model exists already in the `die()` function implementation. I will add the other change.

Reply:
that is a very long document.  i added anchors for relevant sections. 
'fromlist-ref' 
'revert'

Reply:
Right, I will just add all the links in the `die()` one though, so that they are all contained in one place